_posts/2019-05-01-terrence-lee.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title:  "Terence Lee"
+---
+Terence leads Heroku’s Ruby Task Force curating the Ruby experience on the platform. He has worked on several open source software projects such as Ruby (itself), mruby, mruby-cli, Bundler, Resque. When he is not contributing to OSS projects he is helping with the Rails Girls movement. Terence likes to get together with people so you will probably meet him at the one or other awesome Ruby event. At least once there will be a karaoke session with him. For this occasion, he created the [#rubykaraoke](https://twitter.com/hashtag/rubykaraoke?src=hash) Twitter hashtag. When not traveling the world to attend conferences, Terence lives in Austin, TX, the taco capital of America.
+
+You can find Terence on Twitter as [@hone02](https://twitter.com/hone02) and on GitHub as [hone](https://github.com/hone).
```

Comment:
Please add a link to our ticket sales https://ti.to/ruby-unconf/2019

"Buy a ticket to attend RubyUnconf, become a potential speaker and see Terence live on stage"